src/utils/ProteomicsLFQ.cpp

```diff
@@ -878,7 +909,9 @@ class ProteomicsLFQ :
         auto range = transfered_ids.equal_range(fraction_group - 1);
         for (auto& it = range.first; it != range.second; ++it)
         {
-          peptide_ids.push_back(it->second);
+           PeptideIdentification trans = it->second;
```

Comment:
Can you at least add a note, that this will be a problem for some algorithms
if someone used merged IdXMLs as input to proteomicsLFQ?

Reply:
We probably should check that in the beginning then, as well.

Reply:
first test was already there but I added the second one as well